scripts/test_all.sh

```diff
@@ -28,11 +28,11 @@ echo_and_run bazel run @nodejs//:yarn
 printf "\n\nBuilding all targets\n"
 echo_and_run bazel build ...
 
-printf "\n\nTesting all targets\n"
+printf "\n\nTesting all targets (except integration tests)\n"
```

Comment:
why do we still have this shell script?

Reply:
It's useful to make sure that all assertions in CI are easy to run locally. For most purposes the following will do locally:
```
yarn test
yarn test_e2e
yarn test_examples
```